catalog/spatial/registry/registry-federation-admin-service-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -13,20 +13,57 @@
            xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
 
-    <bean id="federationAdminService" class="org.codice.ddf.registry.federationadmin.service.impl.FederationAdminServiceImpl" init-method="init">
+    <bean id="federationAdminService"
+          class="org.codice.ddf.registry.federationadmin.service.impl.FederationAdminServiceImpl"
+          init-method="init">
         <property name="catalogFramework" ref="catalogFramework"/>
-        <property name="registryTransformer" ref="inputTransformer"/>
-        <property name="parser" ref="xmlParser"/>
         <property name="filterBuilder" ref="filterBuilder"/>
+        <property name="parser" ref="xmlParser"/>
+        <property name="registryTransformer" ref="inputTransformer"/>
     </bean>
 
-    <service ref="federationAdminService" interface="org.codice.ddf.registry.federationadmin.service.FederationAdminService"/>
+    <bean id="RefreshRegistrySubscriptions"
+          class="org.codice.ddf.registry.federationadmin.service.impl.RefreshRegistrySubscriptions">
+        <property name="federationAdminService" ref="federationAdminService"/>
+    </bean>
+
+
+    <cm:property-placeholder persistent-id="Registry_Federation_Admin_Service"
+                             update-strategy="reload">
+        <cm:default-properties>
+            <cm:property name="registrySubPollerInterval" value="30"/>
+        </cm:default-properties>
+    </cm:property-placeholder>
+
+
+    <reference-list id="fasRegistryStore" interface="org.codice.ddf.registry.api.RegistryStore"
+                    availability="optional">
+        <reference-listener bind-method="bindRegistryStore" unbind-method="unbindRegistryStore"
+                            ref="RefreshRegistrySubscription"/>
+    </reference-list>
+
+
+    <reference id="filterAdapter" interface="ddf.catalog.filter.FilterAdapter"/>
+
+
+    <service ref="federationAdminService"
+             interface="org.codice.ddf.registry.federationadmin.service.FederationAdminService"/>
 
     <reference id="catalogFramework" interface="ddf.catalog.CatalogFramework"/>
-    <reference id="inputTransformer" interface="ddf.catalog.transform.InputTransformer" filter="(id=rim:RegistryPackage)"/>
+    <reference id="inputTransformer" interface="ddf.catalog.transform.InputTransformer"
+               filter="(id=rim:RegistryPackage)"/>
     <reference id="xmlParser" interface="org.codice.ddf.parser.Parser" filter="(id=xml)"/>
     <reference id="filterBuilder" interface="ddf.catalog.filter.FilterBuilder"/>
 
+    <camelContext xmlns="http://camel.apache.org/schema/blueprint" id="ctxRegistrySubPoller">
+        <route>
+            <from uri="timer://registrySubTimer?fixedRate=true&amp;period={{registrySubPollerInterval}}s"/>
+            <!--change class name -->
+            <to uri="bean:RefreshRegistrySubscription?method=refreshRegistrySubscriptions"/>
```

Comment:
this should be plural  `refreshRegistrySubscriptions` instead of `RefreshRegistrySubscription`